frontend/apps/DatasetDetail.vue

```diff
@@ -58,6 +58,10 @@ const emit = defineEmits([
     'delete:surface'
 ])
 
+const attachmentCount = ref(0); // Number of attachments
```

Comment:
Why do you need this? You can just ask for the number of attachments through `attachments.length`